Configuration/Geometry/python/GeometryRecoDB_cff.py

```diff
@@ -16,9 +16,9 @@
 from Geometry.CaloEventSetup.EcalTrigTowerConstituents_cfi import *
 from Geometry.EcalMapping.EcalMapping_cfi import *
 from Geometry.EcalMapping.EcalMappingRecord_cfi import *
+from Geometry.HcalCommonData.hcalDDConstants_cff import *
```

Comment:
I think the hcalDDConstants_cff.py is already there in the IB of 2015-07-14-1100. I started from there.
________________________________
From: Ianna Osborne [notifications@github.com]
Sent: 22 July 2015 12:10
To: cms-sw/cmssw
Cc: Sunanda Banerjee
Subject: Re: [cmssw] bsunanda:Run2-hcs23 Modify to add the adequate ESProducers in the configuration scenarios (#10267)


In Configuration/Geometry/python/GeometryRecoDB_cff.py<https://github.com/cms-sw/cmssw/pull/10267#discussion_r35198365>:

> @@ -16,9 +16,9 @@
>  from Geometry.CaloEventSetup.EcalTrigTowerConstituents_cfi import *
>  from Geometry.EcalMapping.EcalMapping_cfi import *
>  from Geometry.EcalMapping.EcalMappingRecord_cfi import *
> +from Geometry.HcalCommonData.hcalDDConstants_cff import *


@bsunanda<https://github.com/bsunanda> - hcalParameters_cfi should not be included

—
Reply to this email directly or view it on GitHub<https://github.com/cms-sw/cmssw/pull/10267/files#r35198365>.